### ISSUE TITLE
build actions take a lot of minutes

### ISSUE THREAD

**Issue by mgrider** [kurbos/bevy-shell-template#44]
I cloned this repository, and it promptly used up all my github action minutes. Yes, ALL of them.

Here's a usage report for my account:

<img width="1093" alt="Screenshot 2023-02-09 at 2 18 48 PM" src="https://user-images.githubusercontent.com/604867/217928552-74f54650-2c0d-4800-b46c-1faf1673c88b.png">

It seems ridiculous that so many minutes would have been used for any of the builds. This is clearly a problem with the template.

**Comment by simbleau** [comment]
Yep, this is expected currently. It does testing on all devices in several stages. Very beefy CI. Feel free to slim it down, but the heart of the issue is that we aren't providing a cheffed docker image with dependencies already baked in, like Bevy and Trunk. Installing those takes (very) long and eats up a lot of resources. 

I'd love to see a custom Docker image made. Until then, I'm your best bet is to turn off minute hogs like MacOS CI.

**Comment by mgrider** [comment]
I think a warning on the install instructions is the least you could do here.
